### DISCUSSION TITLE
blocky_error_total	Counter for internal errors

### DISCUSSION THREAD

**Discussion by dynek** [0xERR0R/blocky#1116] [General]
Hello,

Can you please clarify what `blocky_error_total` (Counter for internal errors) is about ?

I mean, if # of errors increase, is there something I should pay attention to?

Thank you

**Comment by dynek** [comment]
I checked once again and I'm unable to understand what they are about, apart from "internal errors".

**Comment by ThinkChaos** [comment]
It's the total number of resolver/query errors ([code](https://github.com/0xERR0R/blocky/blob/v0.22/resolver/metrics_resolver.go#L46-L47)). I agree the docs could be clarified, let me know if you think this helps: https://github.com/0xERR0R/blocky/pull/1270

**Comment by dynek** [comment]
@ThinkChaos that's perfect! Thank you 👍
